mne/io/fieldtrip.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: UTF-8 -*-
+# Copyright (c) 2018, Thomas Hartmann & Dirk Gütlin
+# Authors: Thomas Hartmann <thomas.hartmann@th-ht.de>
+#          Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+# License: BSD (3-clause)
+
+import mne
+import numpy as np
+from ..externals import pymatreader
+
+
+def read_raw_ft(ft_structure_path, data_name='data'):
```

Comment:
you can use https://pypi.org/project/pep257/ on this file to check the docstring formatting.

you should also use `make docstring` from the Makefile in the root folder to check them.